app/views/static_pages/about.html.erb

```diff
@@ -0,0 +1,14 @@
+
+<% provide :title, "About" %>
+<h1>About</h1>
+<p>
+The <a href="https://www.railstutorial.org/"><em>Ruby on Rails
```

Comment:
using i18n for text
change <a></a> by using link_to helper method

Reply:
![image](https://user-images.githubusercontent.com/45727724/89767419-79feb100-db2c-11ea-9914-4320ac1beb3d.png)
 like this?